src/biblebot/bot.py

```
@@ -71,43 +192,76 @@ def load_environment(config_path):
-            if response.status == 200:
-                return await response.json()
-            return None
+    try:
+        async with aiohttp.ClientSession() as session:
+            async with session.get(url, headers=headers, params=params) as response:
+                if response.status == 200:
+                    return await response.json()
+                logger.warning(f"HTTP {response.status} fetching {url}")
+                return None
+    except aiohttp.ClientError as e:
+        logger.error(f"Network error fetching {url}: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)